sopel/modules/admin.py

```diff
@@ -247,61 +268,85 @@ def mode(bot, trigger):
     bot.write(('MODE', bot.nick + ' ' + mode))
 
 
-@sopel.module.require_privmsg("This command only works as a private message.")
-@sopel.module.require_admin("This command requires admin privileges.")
-@sopel.module.commands('set')
-@sopel.module.example('.set core.owner Me')
-def set_config(bot, trigger):
-    """See and modify values of Sopel's config object.
-
-    Trigger args:
-        arg1 - section and option, in the form "section.option"
-        arg2 - value
+def parse_section_option_value(config, trigger):
+    """Parse trigger for set/unset to get relevant config elements.
 
-    If there is no section, section will default to "core".
-    If value is None, the option will be deleted.
+    Returns:
```

Comment:
You could use the Sphinx directive here:

```
:param config: Sopel's config
:param trigger: IRC line trigger
:return: A tuple with ``(section, section_name, static_sec, option, value)``
:raises InvalidSection: section does not exist
:raises InvalidSectionOption: option does not exist for section

The ``value`` is optional and can be returned as ``None`` if omitted from command.
```

Reply:
I am very new to sphinx, so I defer to your expertise here. It has been added as you suggested :smile: